packages/docs/docs/components/code/index.mdx

```diff
@@ -724,61 +724,105 @@ function hello() {
 });
 ```
 
-## Multiple Languages
+## Custom Themes
 
-You can register parsers for multiple different languages by giving them a
-unique name during registration:
+`LezerHighlighter` uses CodeMirror's
+[`HighlightStyle`](https://codemirror.net/examples/styling/) to assign colors to
+specific code tokens. By default, the
+[`DefaultHighlightStyle`](/api/2d/code#DefaultHighlightStyle) is used. You can
+specify your own style by passing it as the second argument to the
+`LezerHighlighter` constructor:
 
 ```ts
-import {LezerHighlighter} from '@motion-canvas/2d';
-import {parser as jsParser} from '@lezer/javascript';
-// highlight-next-line
-import {parser as rsParser} from '@lezer/rust';
+import {Code, LezerHighlighter} from '@motion-canvas/2d';
+import {HighlightStyle} from '@codemirror/language';
+import {tags} from '@lezer/highlight';
+import {parser} from '@lezer/javascript';
 
-LezerHighlighter.registerParser(jsParser);
-// highlight-next-line
-LezerHighlighter.registerParser(rsParser, 'rs');
+const MyStyle = HighlightStyle.define([
+  {tag: tags.keyword, color: 'red'},
+  {tag: tags.function(tags.variableName), color: 'yellow'},
+  {tag: tags.number, color: 'blue'},
+  {tag: tags.string, color: 'green'},
+  // ...
+]);
+
+Code.defaultHighlighter = new LezerHighlighter(parser, MyStyle);
 ```
 
-This name can then be passed to the [`dialect`](/api/2d/components/Code#dialect)
-property of a specific `Code` node to use a different parser:
+## Multiple Languages
 
-```tsx editor mode=code
-import {makeScene2D, Code} from '@motion-canvas/2d';
+You can configure highlighters on a per-node basis using the
+[`highlighter`](/api/2d/components/Code#highlighter) property. This will
+override the default highlighter set in the project configuration file:
 
-export default makeScene2D(function* (view) {
-  // Font properties configured for the view are
-  // inherited by all text nodes, including Code:
-  view.fontSize(28).fontFamily('JetBrains Mono');
+```tsx
+import {Code, LezerHighlighter} from '@motion-canvas/2d';
+import {parser} from '@lezer/rust';
 
-  view.add(
-    <>
-      <Code
-        // this node uses the default parser (javascript)
-        offsetX={-1}
-        x={-400}
-        code={`
+const Rust = new LezerHighlighter(parser);
```

Comment:
Maybe name this `RustHighlighter` instead to avoid confusion with the `Rust` node?